pkg/apis/management.cattle.io/v3/pod_security_types.go

```diff
@@ -41,51 +41,3 @@ type PodSecurityAdmissionConfigurationTemplateExemptions struct {
 	RuntimeClasses []string `json:"runtimeClasses"`
 	Namespaces     []string `json:"namespaces"`
 }
-
-func NewPodSecurityAdmissionConfigurationTemplateRestricted() *PodSecurityAdmissionConfigurationTemplate {
```

Comment:
Note: 
Those two functions do not need to be in the API package, so I moved to `pkg/data/management/podadmissionconfigurationtemplate_data.go` where the only place they are called.